src/components/templates/ReactionPage/ReactionPage.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+import { SHOW_EMOJI_IN_REACTION_PAGE } from "settings";
+
+import { messagesToTheBandSelector, reactionsSelector } from "utils/selectors";
+
+import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
+import { useFirestoreConnect } from "hooks/useFirestoreConnect";
+import { useSelector } from "hooks/useSelector";
+import { useRecentVenueUsers } from "hooks/users";
+import { useVenueChat } from "hooks/useVenueChat";
+import { useVenueId } from "hooks/useVenueId";
+
+import UserList from "components/molecules/UserList";
+
+import { ReactionList } from "./ReactionList";
+
+import "./ReactionPage.scss";
+
+const wantedReactionsSelector = SHOW_EMOJI_IN_REACTION_PAGE
+  ? reactionsSelector
+  : messagesToTheBandSelector;
+
+export const ReactionPage: React.FC = () => {
+  const venueId = useVenueId();
+  const { currentVenue } = useConnectCurrentVenueNG(venueId);
+  const { recentVenueUsers } = useRecentVenueUsers();
+  const { messagesToDisplay: venueChatMessages } = useVenueChat();
+
+  // @debt this is very similar to the query in src/hooks/useReactions.ts, but that filters by createdAt > now
+  useFirestoreConnect(
+    currentVenue
+      ? {
+          collection: "experiences",
+          doc: currentVenue.id,
+          subcollections: [{ collection: "reactions" }],
+          orderBy: ["created_at", "desc"],
+          storeAs: "reactions",
+        }
+      : undefined
+  );
+  const reactions = useSelector(wantedReactionsSelector) ?? [];
```

Comment:
[note] While this would probably be good to do at some stage, I didn't want to deal with the specifics/edge cases of useReactions, and how that may impact other areas of the app unknowingly. The risk didn't seem worth the reward in this PR. So this part is OOS.